aptos-move/block-executor/src/txn_last_input_output.rs

```diff
@@ -226,18 +226,30 @@ impl<K: ModulePath, T: TransactionOutput, E: Debug + Send + Clone> TxnLastInputO
     }
 
     /// Returns the total gas, execution gas, io gas and storage gas of the transaction.
-    pub fn fee_statement(&self, txn_idx: TxnIndex) -> Option<FeeStatement> {
+    pub(crate) fn fee_statement(&self, txn_idx: TxnIndex) -> Option<FeeStatement> {
         match &self.outputs[txn_idx as usize]
             .load_full()
             .expect("[BlockSTM]: Execution output must be recorded after execution")
             .output_status
         {
-            ExecutionStatus::Success(output) => Some(output.fee_statement()),
+            ExecutionStatus::Success(output) | ExecutionStatus::SkipRest(output) => {
+                Some(output.fee_statement())
+            },
             _ => None,
         }
     }
 
-    pub fn update_to_skip_rest(&self, txn_idx: TxnIndex) {
+    pub(crate) fn status_to_skip_rest(&self, txn_idx: TxnIndex) -> bool {
```

Comment:
The name is confusing? We allow abort status here.

Reply:
good point, changing to block_truncated_at_idx and adding a comment.